src/aosm/development.md

```diff
@@ -151,7 +151,7 @@ coverage report --include="*/src/aosm/*" --omit="*/src/aosm/azext_aosm/vendored_
 ```
 
 #### Integration tests
-The integration tests are tests which run real azure CLI commands such as `az aosm nsd publish`. When running for the first time in a repository these tests will create a real resource group and deploy real AOSM resources. These resources will be cleaned up after the tests have run. After the first "live" run these tests will be automatically recorded in the `tests/latest/recordings` folder. These recordings record all communication between the CLI and Azure which mean that the next time the test is run it will no longer be run live but instead will be will be run against the recorded responses. This means that the tests will run much faster and will not create any real resources. 
+The integration tests are tests which run real azure CLI commands such as `az aosm nsd publish`. When running for the first time in a repository these tests will create a real resource group in the subscription that is active on your account and deploy real AOSM resources. These resources will be cleaned up after the tests have run. After the first "live" run these tests will be automatically recorded in the `tests/latest/recordings` folder. These recordings record all communication between the CLI and Azure which mean that the next time the test is run it will no longer be run live but instead will be will be run against the recorded responses. This means that the tests will run much faster and will not create any real resources. The recording also does not rely on the knowledge of a subscription and the credentials will be removed from the recordings.
```

Comment:
Can you say what the name of the resource group is please. (Unless the test output tells you in which case no need).

Reply:
Do these tests run as part of any pipeline?  It isn't clear to me whether unit tests are either - I think we need to clarify this in writing.